Content.Shared/Armable/ArmableSystem.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Content.Shared.Examine;
+using Content.Shared.Toggleable;
+using Content.Shared.Item.ItemToggle.Components;
+
+namespace Content.Shared.Armable;
+
+public sealed class ArmableSystem : EntitySystem
+{
+    [Dependency] private readonly SharedAppearanceSystem _appearance = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<ArmableComponent, ExaminedEvent>(OnExamine);
+        SubscribeLocalEvent<ArmableComponent, ItemToggleActivateAttemptEvent>(TryArming);
+        SubscribeLocalEvent<ArmableComponent, ItemToggledEvent>(ArmingDone);
+    }
+
+    private void TryArming(Entity<ArmableComponent> entity, ref ItemToggleActivateAttemptEvent args)
+    {
+        if (TryComp<ItemToggleComponent>(entity, out var comp))
+            comp.Activated = true;
+    }
+
+    private void OnExamine(EntityUid uid, ArmableComponent comp, ExaminedEvent args)
+    {
+        if (!args.IsInDetailsRange || !comp.ShowStatusOnExamination || !TryComp<ItemToggleComponent>(uid, out var itemToggle))
+            return;
+
+        if(itemToggle.Activated)
```

Comment:
You need a space here.

```suggestion
        if (itemToggle.Activated)
```